client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -352,10 +359,14 @@ export default connect(
 				dispatch( fetchReceipt( receiptId ) );
 			},
 			fetchSitePlans( site ) {
-				dispatch( fetchSitePlans( site.ID ) );
+				if ( site ) {
+					dispatch( fetchSitePlans( site.ID ) );
+				}
 			},
 			refreshSitePlans( site ) {
-				dispatch( refreshSitePlans( site.ID ) );
+				if ( site ) {
```

Comment:
Since you touched this part. I think there might be bug in the code that calls this method. See line 112:
```js
this.props.refreshSitePlans( this.props.selectedSite.ID );
```

I don't think ID can have another ID ;)